app/jenkins-for-jira-ui/src/common/util/jenkinsConnectionsUtils.ts

```diff
@@ -12,6 +12,26 @@ export const setName = async (event: React.ChangeEvent<HTMLInputElement>, setSer
 	setServerName(event.target.value);
 };
 
+export const isValid = (
```

Comment:
We already have `isFormValid` below but it's a little verbose in it's handling of setting errors. It's currently being used by the existing flow and didn't want to modify it so created this one instead. Figured it would be better to create a new function and come back and clean up post rollout (todo added to list)